SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -0,0 +1,45 @@
+Summary:        Prebuilt version of ca-certificates package.
+Name:           prebuilt-ca-certificates
+Version:        20200720
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://hg.mozilla.org
+BuildArch:      noarch
+
+%description
+Prebuilt versions of the ca-certificates package with no runtime dependencies.
+
+%package base
+Summary:        Prebuilt version of ca-certificates-base package.
+BuildRequires:  ca-certificates-base
+Conflicts:      ca-certificates
+Conflicts:      ca-certificates-base
+Conflicts:      ca-certificates-microsoft
+
+%description base
+%{summary}
+
+%prep -q
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_datadir}/pki/ca-trust-legacy/
+mkdir -p %{buildroot}%{_sysconfdir}/pki/
+
+install -p -m 644 %{_datadir}/pki/ca-trust-legacy/* %{buildroot}%{_datadir}/pki/ca-trust-legacy/
+install -p -m 644 %{_datadir}/pki/ca-trust-source/* %{buildroot}%{_datadir}/pki/ca-trust-legacy/
+cp -r %{_sysconfdir}/pki/* %{buildroot}%{_sysconfdir}/pki/
+
+%files
+# Base certs bundle file with trust
+%{_datadir}/pki/*
+%{_sysconfdir}/pki/*
```

Comment:
Here's a list of all the files we should need for the prebuilt version. The rest should not be included:
- `%{_sysconfdir}/pki/cert.pem` (it's a link kept for legacy support);
- `%{_sysconfdir}/pki/certs/*` (directory with... more links :));
- `%{_sysconfdir}/pki/ca-trust/extracted/*` **EXCEPT** for the `README` files (there's a few). This is where the actual cert bundle files are kept and that's where the links point to.
- `%{_sysconfdir}/pki/java/cacerts` (yet another link to one of the cert bundles above);
- `%{_datadir}/pki/ca-trust-legacy/*` (other legacy support files).